pkg/detectors/detectors.go

```diff
@@ -59,6 +69,11 @@ type Result struct {
 	StructuredData *detectorspb.StructuredData
 }
 
+// IsVerified checks if a result is guaranteed to be verified.
+func (r Result) IsVerified() bool {
+	return r.Verified && !r.HasVerificationIndeterminate
+}
```

Comment:
This is a stricter definition of verified, which I think loosens the definition for unverified assuming `unverified = !verified`, which I don't think is necessarily the case.

Should we also add a `IsUnverified() bool` method?

Reply:
Yea, that's a valid point. I think we might need that method as well. Thanks.